arm-eventhub/2015-08-01/swagger/EventHub.json

```diff
@@ -0,0 +1,1771 @@
+        },
+        "messageRetentionInDays": {
+          "format": "int64",
+          "type": "integer",
+          "description": "Number of days to retain the events for this Event Hub."
+        },
+        "partitionCount": {
+          "format": "int64",
+          "type": "integer",
+          "description": "Number of partitions created for EventHub."
+        },
+        "partitionIds": {
+          "type": "array",
+          "items": {
+            "type": "integer"
+          },
+          "description": "Current number of shards on the Event Hub."
+        },
+        "status": {
+          "type": "string",
+          "description": "Enumerates the possible values for the status of the EventHub.",
+          "enum": [
+            "Active",
+            "Disabled",
+            "Restoring",
+            "SendDisabled",
+            "ReceiveDisabled",
+            "Creating",
+            "Deleting",
+            "Renaming",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "EntityStatus",
+            "modelAsString": false
+          }
+        },
+        "updatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The exact time the message has been updated."
+        }
+      }
+    },
+    "EventHubResource": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/EventHubProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Description of EventHub Resource."
+    },
+    "EventHubListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EventHubResource"
+          },
+          "description": "Result of the List EventHub operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of EventHub"
+        }
+      },
+      "description": "The response of the List EventHub operation."
+    },
+    "ConsumerGroupCreateOrUpdateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Location of the resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "ARM type of the namespace."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the consumer group."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ConsumerGroupProperties"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "description": "Parameters supplied to the CreateOrUpdate Consumer Group operation."
+    },
+    "ConsumerGroupProperties": {
+      "properties": {
+        "CreatedAt": {
```

Comment:
again property names are pascal case over here whereas they are camel case in other places. make sure things are consistent